CHANGES.md

```diff
@@ -1,5 +1,43 @@
 # Changes
 
+## edge-21.4.4
+
+This edge release further consolidates the control plane by removing the
+`linkerd-controller` deployment and moving the `sp-validator` container into the
+`destination` deployment.
+
+Annotation inheritance has been added so that +all* Linkerd annotations
+on a `Namespace` resource will be inherited by pods within that namespace.
+In addition, the `config.linkerd.io/proxy-await` annotation has been added which
+enables the [linkerd-await](https://github.com/olix0r/linkerd-await)
+functionality, simplifying the implementation of the await behavior.
```

Comment:
We should mention this is enabled by default. Users can disable it by setting this annotation to `disabled`.